components/admin_console/billing/billing_history.tsx

```diff
@@ -28,9 +29,9 @@ const PAGE_LENGTH = 4;
 
 const noBillingHistorySection = (
     <div className='BillingHistory__noHistory'>
-        <img
-            className='BillingHistory__noHistory-graphic'
```

Comment:
Nit: Can we also remove the class in `components/admin_console/billing/billing_history.scss`